app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainActivity.java

```diff
@@ -0,0 +1,94 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.text.format.DateFormat;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports